posthog/templates/overlays.html

```diff
@@ -27,8 +27,8 @@
         document.body.prepend(element)
     }
 </script>
+{% endif %}
 
-{% endif %} {% if not debug %}
```

Comment:
Don't think it makes sense to have Papercups in debug, it just slows things down and requires you to have internet connection to develop properly.

Reply:
Agreed that it's not useful per se, but it is still a UI consideration and without an internet connection it just goes back to not showing up, so should be fine.